openqa-label-known-issues-and-investigate-hook

```diff
@@ -5,7 +5,10 @@ set -eo pipefail
 # "hook script" intended to be called by openQA instances taking a job ID as
 # parameter and forwarding a complete job URL to "openqa-label-known-issues"
 # on stdin and all left unknowns to "openqa-investigate"
-dir=$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)
+dir=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    pwd
+)
```

Comment:
I think this is the only change I really don't like, but apparently one cannot ignore certain lines

Reply:
Yes, but I can also live with it.

Reply:
Maybe this would be better?
```suggestion
dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
```

Reply:
That's worth a try